resources/themes/cura-light/theme.json

```diff
@@ -317,9 +321,11 @@
     },
 
     "sizes": {
+        "base_unit": [1.0, 1.0],
```

Comment:
I still think that this is fairly useless... :confused: Especially now that when resizing this means resizing several buttons in the toolbox but nothing else.

Reply:
I know. But the alternative is to just move what's basically in the code (`unit * x`) into the theme which makes the already massive file even longer and harder to use. Plus then all these arbitrary names need to be thought up. I know they shouldn't be arbitrary in theory but let's be honest: it's pretty much impossible to make more descriptive terms out of all these "spacing lining margin default whatevers"

Reply:
Fixed in 9b556e9